contributed_definitions/NXreflections.nxdl.xml

```diff
@@ -0,0 +1,415 @@
+        reflection 
+      </doc>
+    </field>
+
+    <field name="bby1" type="NX_INT" units="px" minOccurs="1">
+      <attribute name="description">
+        <doc> Describes the dataset </doc>
+      </attribute>
+      <doc> 
+        The upper pixel y position of the bounding box around the recorded
+        reflection 
+      </doc>
+    </field>
+
+    <field name="bbz0" type="NX_INT" units="frames" minOccurs="1">
+      <attribute name="description">
+        <doc> Describes the dataset </doc>
+      </attribute>
+      <doc> 
+        The lower frame number of the bounding box around the recorded
+        reflection 
+      </doc>
+    </field>
+
+    <field name="bbz1" type="NX_INT" units="frames" minOccurs="1">
+      <attribute name="description">
+        <doc> Describes the dataset </doc>
+      </attribute>
+      <doc> 
+        The upper frame number of the bounding box around the recorded
+        reflection 
+      </doc>
+    </field>
+
+    <field name="bkg_mean" type="NX_FLOAT" minOccurs="1">
+      <attribute name="description">
+        <doc> Describes the dataset </doc>
+      </attribute>
+      <doc> 
+        The mean background under the reflection peak 
+      </doc>
+    </field>
+
+    <field name="int_prf_val" type="NX_FLOAT" minOccurs="0">
+      <attribute name="description">
+        <doc> Describes the dataset </doc>
+      </attribute>
+      <doc> 
+        The estimate of the reflection intensity by profile fitting 
+      </doc>
+    </field>
+
+    <field name="int_prf_var" type="NX_FLOAT" minOccurs="0">
+      <attribute name="description">
+        <doc> Describes the dataset </doc>
+      </attribute>
+      <doc> 
+        The variance on the estimate of the reflection intensity by profile
+        fitting 
+      </doc>
+    </field>
+
+    <field name="int_sum_val" type="NX_FLOAT" minOccurs="1">
+      <attribute name="description">
+        <doc> Describes the dataset </doc>
+      </attribute>
+      <doc> The estimate of the reflection intensity by summation </doc>
+    </field>
+
+    <field name="int_sum_var" type="NX_FLOAT" minOccurs="1">
+      <attribute name="description">
+        <doc> Describes the dataset </doc>
+      </attribute>
+      <doc> 
+        The variance on the estimate of the reflection intensity by
+        summation 
+      </doc>
+    </field>
+
+    <field name="lp" type="NX_FLOAT" minOccurs="1">
+      <attribute name="description">
+        <doc> Describes the dataset </doc>
+      </attribute>
+      <doc> 
+        The LP correction factor to be applied to the reflection intensities
+      </doc>
+    </field>
+
+    <field name="prf_cc" type="NX_FLOAT" minOccurs="0">
+      <attribute name="description">
+        <doc> Describes the dataset </doc>
+      </attribute>
+      <doc> 
+        The correlation of the reflection profile with the reference profile
+        used in profile fitting 
+      </doc>
+    </field>
+
```

Comment:
It is not necessary to specify this as an array.  The ``NX_INT`` type applies to both scalar and array content.

Change ``type="HST_ARRAY(NX_INT)"`` to ``type="NX_INT"``